src/utils/useAtomsDebugValue.ts

```diff
@@ -72,6 +72,6 @@ export const useAtomsDebugValue = (options?: Options) => {
     const unsubscribe = devSubscribeStore?.(callback, 2);
     callback();
     return unsubscribe;
-  }, [enabled, store]);
+  }, [enabled, store, deferAtomSetActions]);
```

Comment:
Ooooops, I forgot about that it was in useEffect. Yeah, you are right, we need useRef in this case.

Reply:
That value is always true in the main renderer, so it is fine?

Reply:
Well, it depends on our goal,
but it reads a stale value, which wasn't my intention.